backend/utils/src/main/java/org/eclipse/sw360/attachments/db/RemoteAttachmentDownloader.java

```diff
@@ -50,36 +49,36 @@ public static int retrieveRemoteAttachments(Supplier<HttpClient> httpClient, Str
         AttachmentContentRepository attachmentContentRepository = new AttachmentContentRepository(new DatabaseConnector(httpClient, dbAttachments));
 
         List<AttachmentContent> remoteAttachments = attachmentContentRepository.getOnlyRemoteAttachments();
-        log.info(format("we have %d remote attachments to retrieve", remoteAttachments.size()));
+        log.info("we have {} remote attachments to retrieve", remoteAttachments.size());
```

Comment:
maybe remove the "we have". it is kind of weird

Reply:
In the first 2 or 3 files I modified I adapted the log messages, but then noticed that placeholders are typically not used in the codebase. (Maybe this is one reason for the statement that logging slows down performance.) In this PR I would like to focus on the update of the logging dependency, so these changes are easier to review. The proper use of the logging API deserves a PR on its own.